cns/restserver/util.go

```diff
@@ -384,12 +384,22 @@ func (service *HTTPRestService) getNetworkContainerResponse(
 
 		containerID, exists = service.state.ContainerIDByOrchestratorContext[podInfo.Name()+podInfo.Namespace()]
 
+		ncVersionListResp, err := service.nma.GetNCVersionList(context.TODO())
+		if err != nil {
+			logger.Errorf("failed to get nc version list from nmagent")
```

Comment:
The function does not terminate here.

Reply:
Updated. Terminate it when it needs termination. Some of the case should not terminate though nc version return has error, then I just skip version check and put nc as pending programing from nmagent side.